src/components/ListedSpecies/GeneralListedInformation.jsx

```diff
@@ -0,0 +1,174 @@
+import { InfoIcon } from '@chakra-ui/icons';
+import {
+  Button,
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightElement,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  Select,
+  Stack,
+  Tooltip,
+} from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import { React, useEffect, useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+import CollapsibleSection from '../CollapsibleSection/CollapsibleSection';
+import options from './DropdownOptions';
+import footNotes from './FootNotes';
+import './GeneralListedInformation.css';
+
+// component/section name not final
+const GeneralListedInformation = ({ speciesName }) => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'speciesName' is defined but never used.